eventsourcing_django/management/commands/sync_followers.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+import argparse
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    Mapping,
+    Tuple,
+    Type,
+    TypeVar,
+    cast,
+)
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from eventsourcing.application import Application
+from eventsourcing.system import Follower, Runner, System
+
+TApplication = TypeVar("TApplication", bound=Application)
+
+
+def sync_follower_with_leaders(
+    follower: Follower, leader_names: Iterable[str]
+) -> Dict[str, int]:
+    """Synchronize a follower with its leader apps."""
+    events_counter = {}
+
+    for leader_name in leader_names:
+        # The library isn't telling us how many events it actually processed,
+        # so we first do a dry-run selection of all new notifications.
+        events_counter[leader_name] = 0
+        start = follower.recorder.max_tracking_id(leader_name) + 1
+        for batch in follower.pull_notifications(leader_name, start):
+            events_counter[leader_name] += len(batch)
+
+        follower.pull_and_process(leader_name, start)
+
+    return events_counter
+
+
+def discover_followers() -> Dict[str, Type[TApplication]]:
+    """Get all followers in the system."""
+    return {}  # TODO
```

Comment:
Not sure where this fits in exactly, but (as you have done below) if you have the runner, you can get the followers from `runner.system.followers`.

Reply:
This is an early iteration but basically the TODO here is for point 3 of the task list: to be able to specify where the runner lives instead of inspecting all installed apps.
Or do you think it's not needed at this point?

Reply:
Now implemented.